### ISSUE TITLE
Bug 58075004: [React Native Gallery -> Sketch]: Unable to draw in the canvas region using keyboard.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#617]
Build Details:
OS Version: Dev (OS Build 27871.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.79.0
AT: Keyboard

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘Media' button present in left navigation pane and expand it.
Navigate to 'Sketch' button and activate it.
Now try to draw in the canvas region using keyboard and observe the issue.

Actual Result:

Unable to draw in the canvas region using keyboard.

Note: User is able to draw using mouse.

Expected Result:
User should be able to draw in the canvas region using keyboard also.

User Impact:
Users with motor disabilities who rely on keyboard will be impacted and will not be able to draw using keyboard.

MAS Reference:
[MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)

Trap ID:
[2.1 - Physical Challenge](https://aka.ms/PhysicalChallenge)

"Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by chiaramooney** [comment]
This issue will be a little more tricky to tackle because it will involve editing the react-native-sketch-canvas module. I'm not up-to-date on this module so I'm not sure the code required to do this but my guess is this will have to be on the native side. This will be a trickier bug to resolve. 

Note: The source code for the Paper React Native Gallery begins at the root of the repo rather than inside the New Arch folder.

**Comment by vineethkuttan** [comment]
This belongs to Community Module react-native-sketch-canvas. Which won't be the part of Fabric anytime soon.
